tiledb/bioimg/converters/base.py

```diff
@@ -343,6 +300,53 @@ def tile_to_tiledb(level_tile: Tuple[slice, ...]) -> None:
             groups.register(name=os.path.basename(output_path), **register_kwargs)
 
 
+def _scale(
+    base_uri: str,
+    output_path: str,
+    level_min: int,
+    axes: Axes,
+    attr_dtype: np.dtype,
+    tiles: Mapping[str, int],
+    pyramid_kwargs: Mapping[str, Any],
+) -> List[str]:
+    uris = [base_uri]
+    with tiledb.open(base_uri) as a:
+        scaler = Scaler(a, axes, **pyramid_kwargs)
+
+    level = level_min + 1
+
+    for index, dimension in enumerate(scaler.resolutions):
+        uri = os.path.join(output_path, f"l_{level}.tdb")
+        schema = _get_schema(
+            axes=axes,
+            shape=dimension,
```

Comment:
Confusing/inconsistent naming: 
- The property is called `resolutions`
- Each element of it is called `dimension`
- This is passed to a parameter called `shape`

Please rename them all to indicate they are shapes (e.g. `resolutions` -> `level_shapes`, `dimension -> shape`).